src/js/script.js

```diff
@@ -11,15 +11,21 @@ var colorPicker = new iro.ColorPicker("#picker", {
     borderColor:"#fff",
   });
 
-// here we put the hex value in the box 
+// here we put the hex value in the box  and changing the background as wheel changes the color
 
 const hexbox = document.getElementById("hexbox")
 // for testing purpose 
 console.log(hexbox)
 
+
+
 colorPicker.on(["color:init","color:change"],(color)=>{
-  hexbox.innerHTML = color.hexString})
+  hexbox.innerHTML = color.hexString
+  const divEle = document.getElementById("changecolor")
+  divEle.style.backgroundColor = color.hexString
```

Comment:
This is very nice but not really in the specifications provided in Figma. Maybe ask the product owner if we could implement this?

Reply:
yeah i have implemented this because in PB in cma-7 said that we need to make color wheel interactive so then i make this 
so if PO approves this so we take it otherwise we just remove it no problem

Reply:
just send a gif or a recording in discord later and see if everyone approves

Reply:
Hi, I'm seeing this locally and it looks really nice! I definitely approve it.

Reply:
Okay then! Everything looks good!